MapboxAndroidDemo/proguard-rules.pro

```diff
@@ -40,3 +40,35 @@
 
 # MAS data models that will be serialized/deserialized over Gson
 -keep class com.mapbox.services.api.directionsmatrix.v1.models.** { *; }
+
+# --- GMS ---
+-keep public class com.google.android.gms.* { public *; }
+-dontwarn com.google.android.gms.**
+
+
+# Consumer proguard rules for plugins
```

Comment:
We should remove ProGuard plugins rules after the different plugins get released (👀 https://github.com/mapbox/mapbox-plugins-android/pull/422) because Proguard rules will be properly managed upstream. That is to say, if included upstream no need to add them here again.